resources/lang/en/fieldtypes.php

```diff
@@ -49,6 +49,7 @@
     'code.config.indent_type' => 'Set your preferred type of indentation.',
     'code.config.key_map' => 'Choose preferred set of keyboard shortcuts.',
     'code.config.mode' => 'Choose language for syntax highlighting.',
+    'code.config.rulers' => 'Display rulers at 80 columns and 120 columns.',
```

Comment:
I'm not sure what the typical approach is for adding language support for all the different languages. If you think this PR is worth introducing, let me know if/what I should do in regards to languages.

Reply:
The users of the corresponding languages will contribute it. Adding English is fine, thanks!